content/en/blog/_posts/2022-10-05-current-state-2019-third-party-audit.md

```diff
@@ -0,0 +1,117 @@
+| 31     | Incorrect docker daemon process name in container manager                   | [#81139](https://github.com/kubernetes/kubernetes/issues/81139) | closed, fixed by [#81083](https://github.com/kubernetes/kubernetes/pull/81083)                                                                                                                                                  |
+| 32     | Use standard formats everywhere                                             | [#81140](https://github.com/kubernetes/kubernetes/issues/81140) | Needs a KEP                                                                                                                                                                                                                     |
+| 33     | Superficial health check provides false sense of safety                     | [#81141](https://github.com/kubernetes/kubernetes/issues/81141) | closed, fixed by [#81319](https://github.com/kubernetes/kubernetes/pull/81319)                                                                                                                                                  |
+| 34     | Hardcoded use of insecure gRPC transport                                    | [#81142](https://github.com/kubernetes/kubernetes/issues/81142) | Needs a KEP                                                                                                                                                                                                                     |
+| 35     | Incorrect handling of Retry-After                                           | [#81143](https://github.com/kubernetes/kubernetes/issues/81143) | closed, fixed by [#91048](https://github.com/kubernetes/kubernetes/pull/91048)                                                                                                                                                  |
+| 36     | Incorrect isKernelPid check                                                 | [#81144](https://github.com/kubernetes/kubernetes/issues/81144) | closed, fixed by [#81086](https://github.com/kubernetes/kubernetes/pull/81086)                                                                                                                                                  |
+| 37     | Kubelet supports insecure TLS ciphersuites                                  | [#81145](https://github.com/kubernetes/kubernetes/issues/81145) | closed but fix needed for [#91444](https://github.com/kubernetes/kubernetes/issues/91444) (see [this comment](https://github.com/kubernetes/kubernetes/issues/81145#issuecomment-630291221))                                           |
+
```

Comment:
The following is a non-blocking suggestion.
I would like to highlight [KEP-1933: Defend Against Logging Secrets via Static Analysis ](https://github.com/kubernetes/enhancements/tree/master/keps/sig-security/1933-secret-logging-static-analysis) which was motivated by the third party security audit and contributors that helped implement the KEP

@PurelyApplied , what do you think of including this KEP in this blog and are there any other contributors that helped with implementing KEP 1933

```suggestion
Along with the issues above, the 2019 third party security audit motivated [Kubernetes Enhancement Proposal (KEP) 1933 Defend Against Logging Secrets via Static Analysis] (https://github.com/kubernetes/enhancements/tree/master/keps/sig-security/1933-secret-logging-static-analysis) to prevent exposing secrets to logs with [Patrick Rhomberg](@PurelyApplied) driving the implementation. As a result of this KEP, [`go-flow-levee`](https://github.com/google/go-flow-levee), a taint propagation analysis tool configured to detect logging of secrets, is executed in a [script](https://github.com/kubernetes/kubernetes/blob/master/hack/verify-govet-levee.sh) as a Prow presubmit job. This KEP was introduced in v1.20.0 as an alpha feature, then graduated to beta in v1.21.0, and graduated to stable in v1.23.0. As stable, the analysis runs as a blocking presubmit test. This KEP also helped resolve the following issues from the 2019 third party security audit:
  - [#81114 Bearer tokens are revealed in logs](https://github.com/kubernetes/kubernetes/issues/81114)
  - [#81117 Environment variables expose sensitive data](https://github.com/kubernetes/kubernetes/issues/81117)
  - [#81130 iSCSI volume storage cleartext secrets in logs](https://github.com/kubernetes/kubernetes/issues/81130)

```

Reply:
@reylejano I added a separate sub-section, to cover this as an example of "inspired outcomes"